shell/components/SortableTable/sorting.js

```diff
@@ -94,11 +99,28 @@ export default {
 
   methods: {
     changeSort(sort, desc) {
+      if (this.setSortFn) {
+        const realSort = this.headers.find(header => header.name === sort).sort || '';
+        const realDesc = realSort.includes(':desc') ? !desc : desc;
+        let correctedSort = realSort;
+
+        // ToDo: these if blocks are some hot garbage but they'll have to do for now...
```

Comment:
In this case, there were some specific cases that still broke sorting, I believe because we were somehow referencing fields should be in metadata but we weren't referring to them as such.

The below if block is removing characters I don't need for sorting.

In both cases, the ToDo's here indicate that I'm not going to mark this PR as ready for review until these are cleaned up a bit.

Using '-' to indicate descending order is a cheap optimization for potential sorting being done in the API since using '-' is less verbose in a get query than ":desc" and we know we might run into url length issues given other parameters.